enterprise/frontend/src/metabase-enterprise/advanced_permissions/index.js

```diff
@@ -42,123 +42,129 @@ const BLOCK_PERMISSION_OPTION = {
   iconColor: "danger",
```

Comment:
For all the plugin files:
Hide whitespace diffs from github, all the files are basically just the old code wrapped in a `export const enableXXXPlugin () => { ... }`